content/blog/what-are-webhooks.md

```diff
@@ -0,0 +1,196 @@
+---
+title: What are Webhooks?
+feature_image: what-are-webhooks.png
+post_image: what-are-webhooks.png
+primary_author:
+    name: Adams Adebayo
+    twitter: olodocoder
+primary_tag: Webhooks Library
+tags:
+    - Convoy
+    - Webhooks Library
+featured: false
+description: Curious about webhooks? Dive into this article to understand what webhooks are, how they work, and how they can revolutionize your data integration process.
+published_at: 2023-06-19T17:00:00.000+00:00
+---
+
+# What are Webhooks? A Complete Guide
```

Comment:
@multi-Adams We can remove this title. The blog title should be enough.

Reply:
Noted.

I'll make the necessary changes.

Reply:
done, please review.